### PR TITLE
Made the disabled state for nav more obvious.

### PR DESCRIPTION
**Pull request by arvind0598**
Disabled pointer events and changed cursor to default as per #27380

### CODE REVIEW

**Comment by XhmikosR** [comment]
https://github.com/twbs/bootstrap/blob/v4-dev/.stylelintrc#L40-L250

File: scss/_nav.scss [hunk 1]
```diff
@@ -46,6 +48,8 @@
 
     &.disabled {
       color: $nav-link-disabled-color;
+      pointer-events: none;
+      cursor: default;
```
Comment (mdo) [inline scss/_nav.scss]: These two lines are unnecessary if the base `.nav-link` already gets it from L25-26.

**Review by mdo — DISMISSED** [review]

**Comment by arvind0598** [comment]
@XhmikosR @mdo I think I've done the requested changes. Sorry for the delay.

File: scss/_nav.scss (hunk 1 above)
Comment (arvind0598) [reply scss/_nav.scss]: Done!

**Review by MartijnCuppens — APPROVED** [review]
LGTM 👍

**Pull request merged**